src/components/Modals/UploadButtonGroup.jsx

```diff
@@ -19,22 +20,70 @@ import { useTheme } from '@mui/material/styles';
  * @param {Function} Props.setFile - The set function for handling files
  * @returns {React.JSX.Element} - The UploadButtonGroup Component
  */
+
+const checkCameraAvailability = async () => {
```

Comment:
Considering the function is make specifically for checking webcam availability, think we could update the function name to `checkWebCamAvailability`